src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -75,6 +83,7 @@ JEventProcessorPODIO::JEventProcessorPODIO() {
             "ReconstructedChargedParticleAssociations",
             "ReconstructedSeededChargedParticles",
             "ReconstructedSeededChargedParticleAssociations",
+            // "ReconstructedChargedParticleIDs",
```

Comment:
Do we need to include these? The one-to-many relation from `ReconstructedChargedParticles` points to them already, but I can't remember if this collection needs to be explicitly included in the output.
```suggestion
            "ReconstructedChargedParticleIDs",
```